lib/main.dart

```diff
@@ -123,8 +126,12 @@ class MyAppState extends State<MyApp> {
   }
 
   void onLoad() async {
-    _notificationsLogic.checkPushPermissions();
+    await _appDBService
```

Comment:
this should be moved out. It's still a database service being called from the UI